### PR TITLE
5110: The loading gear and the counter problem on Map Widget Panel

### PR DESCRIPTION
**Pull request by MuhweziDeo**
## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

#5110 

**What is the current behavior?**
The loading gear and the counter is aligned on the bottom bar

**What is the new behavior?**
The loading gear is not aligned whit the counter

## Breaking change

**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/themes/default/less/loaders.less [hunk 1]
```diff
@@ -47,7 +47,7 @@ div#mapstore-globalspinner {
 	.mapstore-circle-loader(12px, 100%, @ms2-color-primary);
 }
 .mapstore-inline-loader {
-    margin: @padding-left-square-md;
+    margin: 8px 8px 0 4px;
```
Comment (offtherailz) [inline web/client/themes/default/less/loaders.less:50]: here you are chaning the global inline loader.
```suggestion
```
This has not effect

File: web/client/themes/default/less/widget.less [hunk 2]
```diff
@@ -219,6 +219,8 @@
                 .catalog-footer {
                     z-index: 100;
                     .shadow-soft-up;
+                    display: flex;
```
Comment (offtherailz) [inline web/client/themes/default/less/widget.less:222]: Please align the loader to the right, and proper center vertically both the text and the loader.
The height of the footer should not change when loader appears.

**Review by offtherailz — CHANGES_REQUESTED** [review]

**Review by offtherailz — CHANGES_REQUESTED** [review]
It works well. 
- [x] padding of 5px for catalog footer is not needed, please remove.
- [x] Please move the inline style in less files.

**Comment by MuhweziDeo** [comment]
> I simplified your changes. @MuhweziDeo please check the changes and do a final test.

@offtherailz This works the same as before just tested it. Thanks

**Review by offtherailz — APPROVED** [review]

**Pull request merged**